src/libopenrave-core/jsonparser/jsonreader.cpp

```diff
@@ -384,45 +451,52 @@ class JSONReader
         return false;
     }
 
-    boost::shared_ptr<const rapidjson::Document> _GetDocumentFromFilename(const std::string& fullFilename, rapidjson::Document::AllocatorType& alloc)
+    boost::shared_ptr<const rapidjson::Document> _GetDocumentFromFilename(const std::string& referenceURI)
```

Comment:
`referenceURI` -> `referenceUri`, remember to follow convention.

Reply:
fixed